modules/local/assembly_stats_summary.nf

```diff
@@ -0,0 +1,19 @@
+process ASSEMBLY_STATS_SUMMARY {
+    label 'process_single'
+
+    container "quay.io/wslh-bioinformatics/pandas@sha256:bf3cb8e5f695cc7c4cf8cc5ab7e7924d1fc4c40dfbe7cb907110e93a7bf6f101"
+
+    input:
+    path("data/*")
```

Comment:
General comment for input using data directory insted of specific files. may be something to think about for future updates.